src/components/Book.css

```diff
@@ -15,11 +15,10 @@
   margin: auto 0 auto 40px;
 }
 
```

Comment:
## Your final design should match Zeplin's design:
- Features that haven’t been implemented yet should be added as a placeholder and does not need to be implemented
- Kindly [Login to Zeplin](https://app.zeplin.io/login) (hint: you will find the credentials in the [sneak peek](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/sneak_peek_v2_0.md) of the project)
- [Zeplin's design](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/images/bookstore.png)
- 

![Capture](https://user-images.githubusercontent.com/63972670/193379481-06243a58-8926-422e-9caa-632551363f6b.PNG)


- Your design:
- 
![Capture](https://user-images.githubusercontent.com/63972670/193379434-ad847502-4e76-4239-aca4-de7ab7926298.PNG)